github_issue_label_guide.md

```diff
@@ -0,0 +1,93 @@
+# Jina Issue Labeling
+
+## Scope
+These guidelines serve as a primary document for explaining how GitHub issues are labeled in Jina repos. For community members reading this document, it should explain how a label was assigned to your issue, and what that label means. For Jina staff, it describes how to assign labels correctly.
```

Comment:
To get a more open source community feeling, I'd suggest use `Jina maintainers` instead of Jina `staff`

Reply:
Done!